packages/components/src/Components/NotAuthorized/NotAuthorized.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon403 from './Icon403';
+import {
+    Title,
+    Button
+} from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+
+const NotAuthorized = ({ serviceName, ...props }) => {
+    let history = useHistory();
+    let button;
+
+    if(history.length) {
+        button = <Button onClick={ history.goBack }>
+            Return to previous page
+        </Button>;
+    } else {
+        button = <Button onClick={ history.push('/') }>
```

Comment:
This will operate only at app level. Meaning if I go from `insights/compliance/systems` to `insights/compliance/systems/UUID` I can go back to `insights/compliance/systems` if you'll use `history.push('/')` it'll redirect you to `insights/compliance/`.

We've had a consensus with UX folks to redirect users to root URL `window.location.href = './'`